src/_locales/de/messages.json

```diff
@@ -24,7 +24,7 @@
         "description": ""
     },
     "popup_special_page_header": {
-        "message": "Nichts zu tun auf dieser Seite",
+        "message": "Nichts zu tun auf dieser Webseite",
```

Comment:
The source text here uses "page" and whenever we have "page" in the source, we translate it as Seite in German. I don't think we should erase the difference between page and site/website in the translation. Let's not make this nor any of the following similar changes of Webseit --> Seit, unless the source text actually says site/website (and not "page").

Reply:
Reversed change to Seite